### DISCUSSION TITLE
The following issues occurred while running the slither

### DISCUSSION THREAD

**Discussion by zhangzone** [crytic/slither#2262] [Q&A]
Hello, I can use Slither normally on the terminal, but when I use Pychram to run the code, I encounter the following problem: ModuleNotFoundError: No module named 'Slither. core' Slither 'is not a package
I used the slither-doctor according to the method you mentioned before, but it didn't solve the problem

![1](https://github.com/crytic/slither/assets/152855330/b107bb24-313f-401a-88cd-cc023b21f416)

![2](https://github.com/crytic/slither/assets/152855330/f8a18225-15e7-431d-8aba-ed23ab67680d)

**Comment by elopez** [comment]
Hi @zhangzone ! It's possible PyCharm is running a different Python installation, like a virtual environment (venv). If that is the case, make sure Slither is installed in it.

If that doesn't help, could you show us how you're running slither?

**Reply by zhangzone** [reply]
I can see on the terminal that the version of Slither is 0.10.0, and it can run normally on the terminal. However, when using PyCharm, an error occurred. The Python version on PyCharm is 3.10. May I ask if Slither is also installed in PyCharm
![Uploading 1.jpg…]()

**Reply by zhangzone** [reply]
When I use Slither, first download it from the terminal according to your steps, and at the same time, I can use Slither on the terminal. Then, I downloaded your code on GitHub and used PyCharm to run the code, which caused this issue

**Reply by elopez** [reply]
you cannot run the code directly from the repo, you must first install it with pip. If you'd like to work on modifying slither, a good option is to use an editable (`-e`) pip installation on a virtual environment (venv). You can follow these instructions to set it up: https://github.com/crytic/slither/wiki/Developer-installation

**Reply by zhangzone** [reply]
think you.when i follow these instructions to set it up: https://github.com/crytic/slither/wiki/Developer-installation,The command source ./env/bin/activate shows that there is no such file or directory. Do I need to install a virtual environment on Ubuntu before setting up

![图片](https://github.com/crytic/slither/assets/152855330/4a37b3e8-782c-408a-b6d9-53d1b4e88d5e)

**Reply by zhangzone** [reply]
think.I have resolved this issue and need to execute apt install python3.10-venv before make dev

**Reply by zhangzone** [reply]
I used your method to solve this problem, but in the end, the same problem still occurred, and I also used other computers to operate in the same situation
![Uploading 截图 2023-12-15 14-26-00.png…]()

**Comment by glarregay-tob** [comment]
I think the issue is that you're trying to import `Slither`, but the package name is `slither` (lowercase). If that doesn't work, check what @elopez suggested.

**Reply by zhangzone** [reply]
The import uses slither, not Slither
